### ISSUE TITLE
Giving 0 points on file upload exercise not possible

### ISSUE THREAD

**Issue by tessaRuckstuhl** [ls1intum/Artemis#1600]
### Describe the bug

When assessing a file upload exercise, the error message "Each Feedback item must have a score." is shown and the submit button is disabled when trying to give 0 points.
#### To Reproduce

Preconditon: File Upload Exercise to assess has been created
1. Go to Tutor Course Dashboard
2. Click on the to be assessed file upload exercise
3. Start the assessment
4. Click Add new Feedback
5. See error

#### Expected behavior

I would expect, that it is possible to give 0 points on file upload exercises.
#### Screenshots

![image](https://user-images.githubusercontent.com/50466283/84058961-2cde5180-a9ba-11ea-8dba-9e87927c082d.png)

### Environment

<details><pre>
 - OS: Windows
 - Browser Chrome
 - Version 83.0.4103.61
</pre></details>

#### Output in Web Console

<details><pre>
  [REPLACE THIS WITH YOUR INFORMATION]
</pre></details>

**Comment by tessaRuckstuhl** [comment]
Yes tried that. That's actually what I did in the screenshot above. Tried adding multiple feedback box but nothing helps.

**Comment by MaisiKoleni** [comment]
I tried it out just now, I can enter some feedback in the "General Feedback" section and submit, no other feedback blocks must be present. Also using Win 10 and a Chromium-based Browser.

Does this really not work for you? I would be awesome if you could share how Artemis reacts to that, e.g. by providing screenshot or an error message. (You already did that, but in that case a feedback block is present)

**Comment by tessaRuckstuhl** [comment]
Sorry for just getting back to you now. You're correct, it works when **only** entering feedback in the "General Feedback".  I'd say the issue mentioned above is anyway problematic though - but regarding that there is this workaround probably low priority.
